stored_queries/GO/GO_get_terms_from_ws_feature.yaml

```diff
@@ -0,0 +1,39 @@
+# Get the terms from a feature
+
+name: GO_get_terms_from_feature
+params:
+  type: object
+  required: [feature_id, ts]
+  properties:
+    obj_ref:
+      type: string
+      title: Workspace versioned object reference
```

Comment:
this should be a feature reference now

Reply:
what am I saying? - the reference for a genome feature will need to include the object id and the feature id, right? the feature id alone is ambiguous.